### DISCUSSION TITLE
Allow members of a "parent" Challenge to join automatically underlying opportunities

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#595] [Ideas] [answered]
For me this would be a welcome feature. When members join the challenge they are automatically joining underlying opportunities. 
Now the administrator (me) needs to join them individually.

**Comment by techsmyth** [comment, marked as answer]
Hi Gerlof,

I have captured this in the following epic: https://github.com/alkem-io/alkemio/issues/616

The current work on Community model will help, but still there would be work / testing involved. Suggest we talk this through further at the next touch base to judge relative priority.

Kind regards,

Neil
